mods/sbz_meteorites/attractor.lua

```diff
@@ -1,9 +1,10 @@
 local elapsed = 0
+local attractiveItem = "sbz_meteorites:neutronium" -- temporary and subject to change
 
 local function attract_meteorites(pos, dtime, t)
     elapsed = elapsed + dtime
     for _, obj in ipairs(minetest.get_objects_inside_radius(pos, 200)) do
-        if not obj:is_player() and obj:get_luaentity() and obj:get_luaentity().name == "sbz_meteorites:meteorite" then
+        if (obj:is_player() and obj:get_wielded_item():to_string() == attractiveItem) or (obj:get_luaentity() and obj:get_luaentity().name == "sbz_meteorites:meteorite") then
```

Comment:
if you have more than 1 neutronium it wont work, you need to use get_name, or maybe you can even make the attraction/repulsion stronger depending on how much neutronium you are wielding

After you fix this, this may get merged, zander likes the idea

Reply:
yeah, I think I can make it stronger depending on neutronium count, maybe reversed if antineutronium

Reply:
dont make it too strong unless its funny, if its not or if its laggy then yea limit it

Reply:
done